overlord/ifacestate/handlers.go

```diff
@@ -346,7 +377,7 @@ func (m *InterfaceManager) doConnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	conns, err := getConns(st)
+	connections, err := getConns(st)
```

Comment:
Change seems unwanted considering everywhere else continues to refer to this specific map as conns, including the function itself at the right-hand side and other touched areas in the patch below.